pages/research/contents.json.js

```diff
@@ -0,0 +1,30 @@
+import contents from '../../contents'
+
+function Contents() {
+  // getServerSideProps will do the heavy lifting
+}
+
+export function getServerSideProps({ res }) {
+  const pages = contents.articles
+    .concat(contents.tools)
+    .concat(contents.extras)
+    .map(({ date, id }) => {
+      const result = { page: `research/${id}` }
```

Comment:
these are supposed to be partial URLs? no reason to think otherwise, just confirming

Reply:
oh nevermind, that will happen at the `sitemap` generation stage, doesn't need to be done here